spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -396,14 +395,12 @@ public interface EventMetadata {
     public Set<Annotation> getQualifiers();
     public InjectionPoint getInjectionPoint();
     public Type getType();
-    public boolean isAsync();
```

Comment:
Because you know if you are in a synchronous or asynchronous observer, therefore you know that Event was fired asynchronously or synchronously since there's no more way to trigger async observer with sync event.